File: 1678-goal-parser-interpretation/1678-goal-parser-interpretation.cpp
class Solution {
public:
    string interpret(string command) {
        string temp = "";
        for(int i = 0; i < command.size(); i++)
        {
            if(command[i] == '(' && command[i+1] == ')')
            {
                temp+="o";
            }
            else if(command[i] == '(' || command[i] == ')')
                continue;
            else
                temp+=command[i];
        }
        return temp;
    }
};

File: 1678-goal-parser-interpretation/1678-goal-parser-interpretation.js
/**
 * @param {string} command
 * @return {string}
 */
var interpret = function(command) {
    let res = ""
    for(let i = 0; i < command.length; i++)
    {
        if(command[i] === "(" && command[i+1] === ")")
        {
            res+="o";
        }
        else if(command[i] === "(" || command[i] === ")")
        {
            continue;
        }
        else{
            res+=command[i]
        }
    }
    return res;
};